src/test/java/com/google/sps/MatchServletTest.java

```diff
@@ -0,0 +1,75 @@
+package com.google.sps;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockito.Mockito.*;
```

Comment:
maybe just include the classes that you need? (https://google.github.io/styleguide/javaguide.html#s3.3.1-wildcard-imports)

Reply:
done